src/commands/typescript/update.ts

```diff
@@ -67,25 +79,48 @@ export default class Update extends SfdxCommand {
     await fs.writeJson(tsConfigPath, tsConfig);
   }
 
+  private async updateEsTarget(): Promise<void> {
```

Comment:
You can simplify this method now that `location` is available on the `SinglePacakgeRepo` class:

```typescript
private async updateEsTarget(): Promise<void> {
  for (const pkg of this.packages) {
    await this.updatedEsTargetConfig(pkg.location);
  }
}
```

Reply:
Much cleaner thanks !